internal/config/config.go

```diff
@@ -137,13 +140,11 @@ func (c *AuthConfig) Token(hostname string) (string, string) {
 	return token, source
 }
 
-// TokenFromEnvOrConfig will retrieve the auth token for the given hostname,
-// searching environment variables, and plain text config.
-func (c *AuthConfig) TokenFromEnvOrConfig(hostname string) (string, string) {
-	if c.tokenOverride != nil {
-		return c.tokenOverride(hostname)
-	}
-	return ghAuth.TokenFromEnvOrConfig(hostname)
+// HasEnterpriseEnvToken checks whether the current env or config contains an enterprise token
+func (c *AuthConfig) HasEnterpriseEnvToken() bool {
+	token, source := ghAuth.TokenFromEnvOrConfig("")
```

Comment:
This works right now, but only because of an accident. We just so happen to interpret an empty string passed to TokenFromEnvOrConfig as an "enterprise" hostname because it technically doesn't match `github.com`. We should add a test to go-gh to make note that we shouldn't accidentally break this behavior. But this shouldn't prevent this PR from shipping 👍

Reply:
would you prefer I use `example.com` as before?